scripts/generate-abis.ts

```diff
@@ -56,9 +56,22 @@ function main() {
       const fileName = `${contractName}.abi.ts`;
       const filePath = path.join(versionOutputDir, fileName);
 
-      const content = `// This file is auto-generated by scripts/generate-abis.ts
-export default ${JSON.stringify(abi, null, 2)} as const;`;
-      fs.writeFileSync(filePath, content);
+      // It is generally better to use the Steam API for larger files
```

Comment:
nitpick: Stream API

Reply:
Fixed in 0e7c2b494c9f3084fcf805103fc9f43a8920658a.